File: src/targets/ruby/native/fixtures/cookies.rb
require 'uri'
require 'net/http'

<<<<<<< HEAD
url = URI("https://httpbin.org/cookies")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Get.new(url)
=======
url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
>>>>>>> upstream/master
request["cookie"] = 'foo=bar; bar=baz'

response = http.request(request)
puts response.read_body

File: src/targets/node/native/fixtures/cookies.js
<<<<<<< HEAD
const http = require('https');

const options = {
  method: 'GET',
  hostname: 'httpbin.org',
  port: null,
  path: '/cookies',
=======
const http = require('http');

const options = {
  method: 'POST',
  hostname: 'mockbin.com',
  port: null,
  path: '/har',
>>>>>>> upstream/master
  headers: {
    cookie: 'foo=bar; bar=baz'
  }
};

const req = http.request(options, function (res) {
  const chunks = [];

  res.on('data', function (chunk) {
    chunks.push(chunk);
  });

  res.on('end', function () {
    const body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();